spec/models/claim_spec.rb

```diff
@@ -360,12 +360,13 @@
     end
   end
 
-  describe "submitted" do
+  describe "awaiting_approval" do
     let!(:submitted_claims) { create_list(:claim, 5, :submitted) }
     let!(:unsubmitted_claims) { create_list(:claim, 2, :submittable) }
+    let!(:approved_claims) { create_list(:claim, 5, :submitted, approved_at: Time.zone.now) }
 
-    it "returns submitted claims" do
-      expect(subject.class.submitted).to match_array(submitted_claims)
+    it "returns claims awaiting approval" do
```

Comment:
I think we talk about "awaiting approval", but also use the "submitted" trait and variable name. Maybe we should included the word submitted in the spec wording to join things up a bit more? Either that or we rename the trait, but that feels a bit more involved and may not make sense elsewhere.

```suggestion
    it "returns submitted claims awaiting approval" do
```